values.yaml

```diff
@@ -21,6 +21,12 @@ server:
   #     memory: 256Mi
   #     cpu: 250m
 
+  # authDelegator enables a cluster role binding to be attached to the service 
+  # account.  This cluster role binding can be used to setup Kubernetes auth 
+  # method.  https://www.vaultproject.io/docs/auth/kubernetes.html
+  authDelegator:
+    enabled: true
```

Comment:
Are we sure we want to default this to true? I think this should be an opt-in feature.

Reply:
Agreed.  Changed in https://github.com/hashicorp/vault-helm/pull/10/commits/b82db908dd2bebbbcb158824e05ea3ca1e3b7ef4